app/services/idv/proofer.rb

```diff
@@ -1,75 +1,39 @@
 module Idv
   module Proofer
-    ATTRIBUTES = %i[
-      uuid
-      first_name last_name middle_name gen
-      address1 address2 city state zipcode
-      prev_address1 prev_address2 prev_city prev_state prev_zipcode
-      ssn dob phone email
-      state_id_number state_id_type state_id_jurisdiction
-    ].freeze
-
-    STAGES = %i[resolution state_id address].freeze
-
     @vendors = nil
 
     class << self
-      def attribute?(key)
-        ATTRIBUTES.include?(key&.to_sym)
-      end
-
-      def get_vendor(stage)
-        stage = stage.to_sym
-        vendor = vendors[stage]
-        return vendor if vendor.present?
-        return unless mock_fallback_enabled?
-        mock_vendors[stage]
-      end
-
       def validate_vendors!
-        return if mock_fallback_enabled?
-        missing_stages = STAGES - vendors.keys
-        return if missing_stages.empty?
-        raise "No proofer vendor configured for stage(s): #{missing_stages.join(', ')}"
+        resolution_vendor.new
```

Comment:
These calls should fail if mocks are disabled but LexisNexis/AAMVA are not defined